pkg/api/helpers/download.go

```diff
@@ -0,0 +1,203 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/go-openapi/swag"
+	"github.com/treeverse/lakefs/pkg/api/apigen"
+	"github.com/treeverse/lakefs/pkg/uri"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	DefaultDownloadPartSize    int64 = 1024 * 1024 * 8 // 8MB
```

Comment:
@nopcoder Don't you think we should add some way for the user to control it?
We can add another task for it if you don't think we should do it here